index.html

```diff
@@ -1374,6 +1449,20 @@ <h3>
           developer's preferred <a>display mode</a> for the web application.
         </p>
       </section>
+      <section>
+        <h3>
+          <code>display_override</code> member
```

Comment:
You also need to add an entry for `display_override` in the Web App Manifest WebIDL (search for `dictionary WebAppManifest`).